src/perception/vision_pipeline/vision_pipeline/node_zed.py

```diff
@@ -0,0 +1,51 @@
+# import ROS2 libraries
```

Comment:
call this `node_zed_simulator.py` or something, node_zed is not very descriptive